dcos/servicemanager.py

```diff
@@ -39,4 +45,34 @@ def start_service(self, package_name, package_version, options):
             json['packageVersion'] = package_version
         if options is not None:
             json['options'] = options
-        return self.cosmos.call_endpoint(endpoint, json=json)
+        try:
+            return self.cosmos.call_endpoint(endpoint, json=json)
+        except DCOSAuthenticationException:
+            raise
+        except DCOSAuthorizationException:
+            raise
```

Comment:
But it would be even nicer if those cases could be handled by the `@cosmos_error` decorator.